security/pkg/k8s/chiron/controller_test.go

```diff
@@ -15,12 +15,53 @@
 package chiron
 
 import (
+	"bytes"
+	"io/ioutil"
+	"reflect"
 	"testing"
 	"time"
 
+	"k8s.io/api/admissionregistration/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+
+	"istio.io/istio/security/pkg/pki/util"
+
+	cert "k8s.io/api/certificates/v1beta1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sRuntime "k8s.io/apimachinery/pkg/runtime"
+
 	"k8s.io/client-go/kubernetes/fake"
 )
 
+const (
+	exampleCACert1 = `-----BEGIN CERTIFICATE-----
```

Comment:
Please indicate a command that can be made to regenerate these and make sure its valid for at least 5 years - otherwise it will expire and will be a huge pain like has happened over and over

Reply:
Done.